### PR TITLE
Translate to portuguese.

### PR DESCRIPTION
**Pull request by edsiddos**
Correcting indentation problems of source code. Translate to Portuguese.

### CODE REVIEW

**Comment by ravage84** [comment]
Corrected version of https://github.com/cakephp/docs/pull/3697
@edsiddos you don't need to close a Pull Request if there are problems, just add commits to it.

File: pt/tutorials-and-examples/blog-auth-example/auth.rst [hunk 1]
```diff
@@ -1,13 +1,419 @@
...
+coisas mais gerais no AppController para suas próprias regras baseadas em papéis.
+
+Se precisar de mais controle, nós sugerimos que leia o guia completo do Auth
+:doc:`/core-libraries/components/authentication` seção onde você encontrará mais
+sobre a configuração do componente, criação de classes de Autorização customizadas, e muito mais.
+
+Sugerimos as seguintes leituras
+-------------------------------
+
+1. :doc:`/console-and-shells/code-generation-with-bake` Generating basic CRUD code
```
Comment (cake17) [inline pt/tutorials-and-examples/blog-auth-example/auth.rst]: this should be :doc:`/bake/usage` to stick to english version

File: pt/tutorials-and-examples/blog-auth-example/auth.rst [hunk 2]
```diff
@@ -1,13 +1,419 @@
...
+2. :doc:`/core-libraries/components/authentication`: User registration and login
```
Comment (cake17) [inline pt/tutorials-and-examples/blog-auth-example/auth.rst]: Same here with, this should be :doc:`/controllers/components/authentication`

File: pt/tutorials-and-examples/blog-auth-example/auth.rst [hunk 3]
```diff
@@ -1,13 +1,419 @@
...
+        return $this->exists(['id' => $articleId, 'user_id' => $userId]);
+    }
+
+Isso conclui então nossa autorização simples e nosso tutorial de autorização.
+Para garantir o UsersController você pode seguir as mesmas técnicas que usamos
+para ArticlesController, você também pode ser mais criativo e codificar algumas
+coisas mais gerais no AppController para suas próprias regras baseadas em papéis.
+
+Se precisar de mais controle, nós sugerimos que leia o guia completo do Auth
+:doc:`/core-libraries/components/authentication` seção onde você encontrará mais
```
Comment (cake17) [inline pt/tutorials-and-examples/blog-auth-example/auth.rst]: here should be :doc:`/controllers/components/authentication`

**Comment by cake17** [comment]
I think I can merge this and I'll fix the links post-merge

**Pull request merged**